Makefile

```diff
@@ -16,8 +16,10 @@ test-integration: tidy
 
 .PHONY: test-main
 test-main: tidy
-	@$(eval attestation := $(shell cat testdata/nitro_attestation.b64))
-	@go run cmd/nitrite/main.go -attestation $(attestation) > /dev/null
+	@$(eval attestation := $(shell cat internal/testdata/nitro_attestation.b64))
+	@go run cmd/nitrite/main.go -attestation $(attestation) 1>/dev/null
+	@$(eval attestation := $(shell cat internal/testdata/nitro_attestation_debug.b64))
+	@go run cmd/nitrite/main.go -attestation $(attestation) -allowdebug true 1>/dev/null
```

Comment:
Making sure we can verify a debug attestation as well.

Reply:
I added a suggestion for updating the nitrite cmd to allow us to pipe in an attestation string. If you make the update, then you can update these commands to
```make
NITRITE_CMD=./cmd/nitrite/main.go
TESTDATA=./testdata/internal
#...
test-main: tidy
    @cat $(TESTDATA)/nitro_attestation.b64 | go run $(NITRITE_CMD) -attestation -
    @cat $(TESTDATA)/nitro_attestation_debug.b64 | go run $(NITRITE_CMD) -attestation - -allowdebug true
```

Reply:
I took your suggestions for pre-defining the paths, but I left the two step process. That syntax makes it way easier to stay within 80 chars. If you have a nice way of going that using pipe, you're welcome to tune it further.

Reply:
```make
test-main: tidy
	@cat $(TESTDATA)/nitro_attestation.b64 | \
		go run $(NITRITE_CMD) -attestation - 1>/dev/null
	@cat $(TESTDATA)/nitro_attestation_debug.b64 | \
		go run $(NITRITE_CMD) -attestation - -allowdebug true 1>/dev/null
	@echo "ok\tcmd/nitrite/main.go"
```

Reply:
done